Podfile.lock

```diff
@@ -47,7 +47,7 @@ PODS:
   - "GoogleToolboxForMac/NSString+URLArguments (2.2.1)"
   - Gridicons (0.18)
   - GTMSessionFetcher/Core (1.2.2)
-  - Gutenberg (1.10.2):
+  - Gutenberg (1.10.3):
```

Comment:
Not sure how why Podfile.lock has 1.10.3 before we released it?  @etoledom

Reply:
Cocoapods reads the pod version from the Podspec file on `gutenberg-mobile`. And the Podspec gets the version from `package.json`. 

So, as soon as we change the `package.json` version number, cocoapods will know what `gutenberg-mobile` version we are installing 🎉

Reply:
Thanks, @etoledom!